src/components/TasksContainer.js

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'react';
+import Task from './Task';
+import Button from './Button';
+import AddTask from './AddTask';
+const initialTasks = [
+  {
+    id: 1,
+    text: 'Doctors Appointment',
+    day: 'Feb 5th at 2:30pm',
+    reminder: true
+  },
+  {
+    id: 2,
+    text: 'Meeting at School',
+    day: 'Feb 6th at 1:30pm',
+    reminder: true
+  },
+  {
+    id: 3,
+    text: 'food shopping',
+    day: 'Feb 6th at 5:30pm',
+    reminder: false
+  }
+];
+
+const TasksContainer = (props) => {
+  const [tasks, setTasks] = useState(initialTasks);
+  const [showAddTask, setShowAddTask] = useState(false);
+  const onShow = () => {
+    setShowAddTask(!showAddTask);
+  };
+  //add tasks

```

Comment:
Here's a suggestion on how to improve this comment

```suggestion
  // This method creates a new task for the user

```

Reply:
Done